pkg/arvo/sys/lull.hoon

```diff
@@ -803,6 +804,7 @@
         [%kroc bones=(list [ship bone])]
         $>(%plea vane-task)
         [%deep =deep]
+        [%stun =stun]
```

Comment:
For future flexibility, this task should include a ship (or possibly a ship and a lane).